decidim/decidim
# Remove GIF and BMP from the default file formats

#### :tophat: What? Why?
BMP should not be allowed and GIF can cause problems on the website, so let's remove them from the default formats and leave it up to the admins to break their sites.

See #9195 for details.

#### :pushpin: Related Issues
- Fixes #9195

#### Testing
- Create the development app
- See that the file formats are no longer added by default
- Create a new organization in the app
- See that the file formats are no longer added by default
